CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.2
+
+_Sep 8, 2022_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+### `@mui/x-data-grid@v5.17.2` / `@mui/x-data-grid-pro@v5.17.2` / `@mui/x-data-grid-premium@v5.17.2`
+
+#### Changes
+
+- [DataGridPremium] Export column grouping in Excel (#5895) @alexfauquette
+
+### `@mui/x-date-pickers@v5.0.1` / `@mui/x-date-pickers-pro@v5.0.1`
```

Comment:
For v6 we should always release all our stable packages with the same version, but I think that for 1 patch before the alpha it's not worth jumping the pickers to 5.17.2

Reply:
No strong opinion here
I just think bumping from v5.0.0 to v5.17.2 to v6.0.0-alpha.1 can be confusing.
We probably should have bumped to v5.17.0 instead of 5.0.0 last week.

Reply:
> We probably should have bumped to v5.17.0 instead of 5.0.0 last week.

Yes, that's my thinking as well, if we did not align the version with the "stable" release - no point in doing that now.

Reply:
I was about to leave a new comment until I found this thread. I agree with the conclusion 👍. If we need to release more versions, it will be clearer with a `@mui/x-date-pickers-pro@v5.17.2`. So it can look like this in people's `package.json`:

```json
  "@mui/x-data-grid": "5.17.2",
  "@mui/x-date-pickers": "5.17.2",
```